src/pretalx/agenda/views/talk.py

```diff
@@ -66,14 +69,14 @@ class TalkView(PermissionRequired, DetailView):
 
     def get_object(self, queryset=None):
         with suppress(AttributeError, TalkSlot.DoesNotExist):
-            return self.request.event.current_schedule.talks.get(
+            return self.request.event.current_schedule.talks.filter(
```

Comment:
Wouldn't it be better to make the submission itself the object of the template and view, now that we potentially have multiple slots?